### PR TITLE
YLP-743 A patient team invitation requires that the patient account already exists

### PR DESCRIPTION
**Pull request by franck-fourel**
based on PR #145

### CODE REVIEW

File: api/invite_test.go [hunk 1]
```diff
@@ -564,112 +577,112 @@ func TestDismissInvite_NoPerms(t *testing.T) {
...
-			},
-		},
-		{
-			desc:     "delete the other invitation we sent",
-			method:   http.MethodPut,
-			url:      fmt.Sprintf("/%s/invited/other@youremail.com", testing_uid1),
-			token:    testing_token_uid1,
-			respCode: http.StatusOK,
-		},
+		// {
```
Comment (Nas84) [inline api/invite_test.go]: Why the test are commented ?

**Review by Nas84 — CHANGES_REQUESTED** [review]

File: api/invite_test.go (hunk 1 above)
Comment (franck-fourel) [reply api/invite_test.go]: reverted

**Review by Nas84 — APPROVED** [review]
looks good to me (of course require the jenkins tests to pass)

**Comment by franck-fourel** [comment]
> looks good to me (of course require the jenkins tests to pass)

can you have a look. 
the failing test was coming from a bad rebase

**Pull request merged**